lms/db/_util.py

```diff
@@ -0,0 +1,24 @@
+from sqlalchemy import Select
+from sqlalchemy.dialects import postgresql
+from sqlalchemy.orm import Query
+
+
+def compile_query(query: Query | Select, literal_binds: bool = True) -> str:
```

Comment:
Might wanna pre-import this function in `make shell` for convenience? https://github.com/hypothesis/lms/blob/041f472a5ab60641887515d0f42df73a8d584174/lms/pshell.py#L30-L47

Reply:
There are several DB-related utility functions that're duplicated in various apps, and this would be another one. We might want to move them into a shared library at some point.

There was a PR a while ago that added a bunch of these to the cookiecutter: https://github.com/hypothesis/cookiecutters/pull/128. We decided not to merge that IIRC because we thought the functions might be better in a library rather than in the cookiecutter, but unless I'm forgetting I don't think we did anything about creating that library.